### PR TITLE
Implement dark mode

### PR DESCRIPTION
**Pull request by rpwgc4**
#Please don't push this until multiple people have reviewed the PR. 

Once these changes are merged into your own branch you will need to redo the migration as per the readme. Colin and Jayson, I may have tweaked a couple lines in Transaction/Index.cshtml, so let me know if you have a conflict on merge.

Summary:
Added an alternate stylesheet, created button in navbar to toggle between stylesheets. This button makes a call which changes a variable in session cookie.

On registration or logout the user's display preference is stored in the database. On login, the stored preference is set as the session value. If a user is logged in the session will time out after 10 minutes of inactivity and store the session style preference.

Actions:
Added darksite.css
Added boolean to ApplicationUser to store user preference
Regenerated the migration to accommodate the latter change
Added session and distributed memory cache services to startup file
Generated Account/Logout page with Identity
Modified login, logout, and registration Razor pages for session logic
Added custom checkbox (toggle button) and implemented stylesheet switch
Implemented Ajax call on button press to update session value
Implemented JS timer to logout user after 10 minutes of inactivity
Implemented session refresh every 5 minutes
Implemented dark mode CSS

### CODE REVIEW

**Review by danfow — COMMENTED** [review]
Not going to pretend I entirely know what's going on with the implementation but I don't see anything that looks out of place here.

File: CommerceBankProject/CommerceBankProject.csproj [hunk 1]
```diff
@@ -18,6 +18,8 @@
     <Compile Remove="Migrations\20210922012928_addmodel.Designer.cs" />
     <Compile Remove="Migrations\20210922015151_addmodel.cs" />
     <Compile Remove="Migrations\20210922015151_addmodel.Designer.cs" />
+    <Compile Remove="Migrations\20210922044612_addModels.cs" />
+    <Compile Remove="Migrations\20210922044612_addModels.Designer.cs" />
   </ItemGroup>
```
Comment (zjwonder) [inline CommerceBankProject/CommerceBankProject.csproj:23]: Is it possible to simplify/get rid of all these compile statements? Seems like they are starting to pile up.

**Review by zjwonder — COMMENTED** [review]
Not a big deal but I was curious about this. I'm interested in seeing the new functionality once we merge it.

**Review by zjwonder — APPROVED** [review]
Super duper

**Pull request merged**